release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2779,6 +2779,38 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.47
+[id="ocp-4-15-47_{context}"]
+=== RHSA-2025:2454 - {product-title} 4.15.47 bug fix and security update
+
+Issued: 12 March 2025
+
+{product-title} release 4.15.47, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2025:2454[RHSA-2025:2454] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2025:2456[RHSA-2025:2456] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.47 --pullspecs
+----
+
+[id="ocp-4-15-47-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, an extra `name` prop was passed into resource list page extensions used to list related operands on the *CSV details* page. This caused the operand list to be filtered by the `CSV` name, which often caused it to be an empty list. With this update, operands are listed as expected. (link:https://issues.redhat.com/browse/OCPBUGS-51332[*OCPBUGS-51332*])
```

Comment:
```suggestion
* Previously, an extra `name` prop was passed into the resource list page extensions used to list related operands on the *CSV details* page. This caused the operand list to be filtered by the `CSV` name, which often caused it to be an empty list. With this update, the operands are listed as expected. (link:https://issues.redhat.com/browse/OCPBUGS-51332[*OCPBUGS-51332*])
```

Reply:
Any idea what a "prop" is? Is it a "property"?

Reply:
@mburke5678 It is a name property like variable name. It is a common term used in react. Here is an example: https://legacy.reactjs.org/warnings/unknown-prop.html
Thanks!